herbert_core/graphics/@IX_dataset_1d/private/plot_oned.m

```diff
@@ -149,6 +149,13 @@
 elseif plot_type(1)=='h'
     plot_histogram (w);
 elseif plot_type(1)=='l'
+    [frac, np] = w.calc_continuous_fraction();
+    if frac<0.5
+        warning('HERBERT:IX_dataset_1D:invalid_argument',...
+            [' Your data contain NaN-s and you are plotting %3.1f%%',...
```

Comment:
data containS. Appreciating that data can sometimes be plural, but here it is a singular container